docs/documentation.html

```diff
@@ -649,6 +761,19 @@ <h3 id="grammar-syntax-and-semantics-parsing-expression-types">Parsing Expressio
     results in an array. The matching is greedy, i.e. the parser tries to match
     the expression as many times as possible. Unlike in regular expressions,
     there is no backtracking.</p>
+
+    <div class="example">
+      <div>
+        <div><em>Example:</em> <code>star = "a"*</code></div>
+        <div><em>Matches:</em> <code>"a"</code>, <code>"aaa"</code></div>
+        <div><em>Does not match:</em> </div>
```

Comment:
May be add an explicitly wording `<nothing (i.e. always match)>` here?
```suggestion
        <div><em>Does not match: &lt;nothing (i.e. always match)&gt;</em> </div>
```

Reply:
Nod, I put `(always matches)`, without quotes, and not in `code` format.